src/core/datastore/persisters/browser/websql-key-value-store-persister.js

```diff
@@ -1,153 +0,0 @@
-import { KinveyError, NotFoundError } from '../../errors';
```

Comment:
This file is not used anywhere, it's an outdated file, left from all the rebasing during the redesign. In case anyone is wondering :)